### PR TITLE
Add remaining changes from #40

### PR DESCRIPTION
**Pull request by jjc2718**
This PR addresses @rando2 's comments from #40 (other than the comments that were addressed in that PR). This includes updating a few figures and cleaning up the abstract and intro a bit.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ Since functional signatures of cancer mutation have been identified across many
...
 
-Across a collection of cancer-associated genetic alterations, RNA sequencing and DNA methylation were the most effective predictors of alteration state.
-Surprisingly, we found that for most alterations, they were approximately equally effective predictors.
-The target gene was the primary driver of performance, rather than the data type, and there was little difference between the top data types for the majority of genes.
-We also found that combining data types into a single multi-omics model often provided little or no improvement in predictive ability over the best individual data type.
+Across a collection of genes recurrently mutated in cancer, RNA sequencing and DNA methylation were the most effective predictors of mutation state.
+Surprisingly, we found that for most mutated genes, they were approximately equally effective predictors.
+Performance was more variable between mutations than it was between data types for the same mutation, and there was little difference between the top data types for the majority of genes.
```
Comment (rando2) [inline content/01.abstract.md:12]: This new version is so much easier for me to follow! I think it communicates the importance of your findings right off the bat!

File: content/04.results.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Each data point represents the mean cross-validated AUPR difference, compared wi
 Significance stars indicate results of Bonferroni-corrected pairwise Wilcoxon tests: \*\*: _p_ < 0.01, \*\*\*: _p_ < 0.001, ns: not statistically significant for a cutoff of _p_ = 0.05.
 **B, C, D.** Volcano-like plots showing mutation presence/absence predictive performance for each gene in each of the 3 gene sets.
 The _x_-axis shows the difference in mean AUPR compared with a baseline model trained on permuted labels, and the _y_-axis shows _p_-values for a paired _t_-test comparing cross-validated AUPR values within folds.
-Points (genes) highlighted in red are overlapping between the Vogelstein gene set and either the random or most mutated gene set.
```
Comment (rando2) [inline content/04.results.md:46]: Now I understand! I was so confused by the two red colors :laughing:

**Review by rando2 — APPROVED** [review]
Looks great to me!

**Comment by jjc2718** [comment]
Here are the new plots:

(fixed boxes in Figure 1B)

figure_1

(changed red dots to X's to indicate overlapping genes)

image

**Review by cgreene — APPROVED** [review]

**Pull request merged**